start.sh

```diff
@@ -100,15 +90,13 @@ docker-compose --compatibility up -d jenkins
 echo "Sleeping for $SLEEP_TIME secs to give jenkins a chance to start up..."
 sleep $SLEEP_TIME
 
-
 # if no apiKey was found, assume it is a fresh install and load fixture data
-if [ -z "$existingPerfanaUserId" ];  then
+# NOTE: if no apiKey was found, it has been created and API_KEY should exist and be not emtpy
+if [[ ! -z $API_KEY ]];  then
     echo "No api keys found Grafana, creating fixture data ..."
     docker-compose --compatibility up -d perfana-fixture
```

Comment:
perfana-fixture lijkt ook een apiKey aan te maken, net zoals dit bash script in regel 55, waardoor mogelijk de apiKey out-of-sync raakt, ik stel voor dat we de apiKey ook aan de perfana-fixture meegeven via docker-compose env en niet meer genereren in fixture zelf